ush/build.sh

```diff
@@ -7,57 +7,46 @@ readonly pkg_root=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )/.."
 
 # User Options
 BUILD_TYPE=${BUILD_TYPE:-"Release"}
-BUILD_DIR=${BUILD_DIR:-"$pkg_root/build"}
+BUILD_DIR=${BUILD_DIR:-"${pkg_root}/build"}
 CMAKE_OPTS=${CMAKE_OPTS:-}
 INSTALL_TARGET=${INSTALL_TARGET:-"wcoss2"}
-INSTALL_PREFIX=${INSTALL_PREFIX:-"$pkg_root/install"}
-
-# By default; build as if NCO is executing this script
-BUILD_4NCO=${BUILD_4NCO:-"YES"}
-
-target=$(echo $INSTALL_TARGET | tr [:upper:] [:lower:])
-if [[ "$target" =~ ^(wcoss2|hera|orion|jet)$ ]]; then
-  source $pkg_root/versions/build.ver
+INSTALL_PREFIX=${INSTALL_PREFIX:-"${pkg_root}/install"}
+
+target="${INSTALL_TARGET,,}"
+if [[ "${target}" =~ ^(wcoss2|hera|orion|jet|hercules)$ ]]; then
+  # prepare the target specific build.ver and run.ver
+  cd "${pkg_root}/versions" || exit 1
+  rm -f build.ver run.ver
+  cp "build.${target}.ver" "build.ver"
+  cp "run.${target}.ver"   "run.ver"
+  cd "${pkg_root}" || exit 1
+  export HOMEobsproc="${pkg_root}"
```

Comment:
Is HOMEobsproc going to point at current package's location?

Reply:
Yes, `HOMEobsproc` will be the clone location for a moment during build before being unset.